CMakeLists.txt

```diff
@@ -43,7 +43,9 @@ target_link_libraries(${APP_TARGET}
     PRIVATE
         mbed-os
         mbed-psa
-        mbed-greentea
+        mbed-unity
+        mbed-utest
+        greentea::client
```

Comment:
We might want to link `greentea::client_userio` and `mbed-greentea-io` if stdio doesn't work

Reply:
Good point, I think it's worth waiting until we've resolved the IO issue.

Reply:
IO issue is resolved and merged https://github.com/ARMmbed/mbed-os/pull/14904

Reply:
Cool. Updated.